### PR TITLE
Convert to Polymer 3, add events type definitions

### PR DESCRIPTION
**Pull request by web-padawan**
Related to #233

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,31 +65,30 @@ To use the Material theme, import the correspondent file from the `theme/materia
...
+1. When in the `vaadin-select` directory, run `npm install` to install dependencies.
 
 1. Run `npm start`, browser will automatically open the component API documentation.
 
-1. You can also open demo or in-browser tests by adding **demo** or **test** to the URL, for example:
+1. You can also open demo or in-browser tests by adding **test** to the URL:
 
-  - http://127.0.0.1:3000/components/vaadin-select/demo
-  - http://127.0.0.1:3000/components/vaadin-select/test
+  - http://127.0.0.1:3000/components/vaadin-select/test/
```
Comment (tomivirkki) [inline README.md:91]: This doesn't seem to work.

File: README.md [hunk 2]
```diff
@@ -99,31 +65,30 @@ To use the Material theme, import the correspondent file from the `theme/materia
...
 
 ## Running demos and tests in a browser
 
 1. Fork the `vaadin-select` repository and clone it locally.
 
-1. Make sure you have [npm](https://www.npmjs.com/) and [Bower](https://bower.io) installed.
+1. Make sure you have [npm](https://www.npmjs.com/) installed.
 
-1. When in the `vaadin-select` directory, run `npm install` and then `bower install` to install dependencies.
+1. When in the `vaadin-select` directory, run `npm install` to install dependencies.
```
Comment (tomivirkki) [inline README.md:85]: For some reason `npm i` doesn't finish successfully

Comment (web-padawan) [reply README.md:85]: Which node version do you have? Is there any error in the console? Everything works for me with Node 12.18.

File: README.md (hunk 1 above)
Comment (web-padawan) [reply README.md]: Thanks, fixed.

Comment (tomivirkki) [reply README.md]: Now the instructions were simply removed. Is there generally some way to run the in-browser tests with the new setup?

Comment (web-padawan) [reply README.md]: Added the note on `npm run debug` script and using manual mode of Web Test Runner.

**Review by tomivirkki — APPROVED** [review]

**Pull request merged**